backend/homework/11/src/main/java/org/example/controller/Controller.java

```diff
@@ -0,0 +1,50 @@
+package org.example.controller;
+
+import org.example.dto.*;
+import org.example.services.MainService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Controller {
+    MainService service;
+    @Autowired
+    public Controller(MainService service)
+    {
+        this.service=service;
+    }
+
+    @PostMapping("/user")
```

Comment:
Make it modular, use better naming convention. Could have used TenantController ....etc